blob/gcsblob/gcsblob.go

```diff
@@ -15,23 +15,6 @@
 // Package gcsblob provides a blob implementation that uses GCS. Use OpenBucket
 // to construct a *blob.Bucket.
 //
-// Open URLs
```

Comment:
Same.

Reply:
In particular, the docstring should explain that by default `OpenBucket` will use default GCP credentials, and link to https://cloud.google.com/docs/authentication/production.